cmd/destroy/destroy.go

```diff
@@ -86,6 +86,7 @@ type destroyCommand struct {
 	nsDestroyer       destroyer
 	secrets           secretHandler
 	k8sClientProvider okteto.K8sClientProvider
+	runner            runnerI
```

Comment:
If we rename the interface, we should rename also this property
```suggestion
	configMapHandler            runnerI
```